kobweb-model/src/main/resources/META-INF/services/org.jetbrains.plugins.gradle.tooling.ModelBuilderService

```diff
@@ -0,0 +1 @@
+com.varabyte.kobweb.intellij.model.gradle.tooling.KobwebModelBuilderService
```

Comment:
Hmmm, I wouldn't document a service file. Nobody will ever look in there for it.

I did add your comments to the KobwebModelBuilderService class directly, though, because they're good points.

I did not mention "in a very hacky way" however. That feels either like a judgment call that needs a citation, or an implementation detail of the way IJ API simply works. Either way, we're the ones choosing to use the API, so it must be useful enough. Let's just use it without commenting on it, unless commenting helps people in the future somehow.

Another way to put it: I don't think every codebase out there which implements the ProjectResolver extension needs to mention it's a hacky API.

Reply:
Ok. The kdoc of the builder is definitely sufficient